From e93c16452562e582f7c75ec9b2183870e51e4891 Mon Sep 17 00:00:00 2001
From: jbettini <ashhxtm@gmail.com>
Date: Fri, 13 Sep 2024 14:34:25 +0200
Subject: [PATCH] Add patch to modify EXTRAVERSION in Makefile for kernel
 version string

This commit updates the EXTRAVERSION field in the Makefile,
append "-thor_kernel" to the kernel version string.

Signed-off-by: jbettini <ashhxtm@gmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 2966d5fbb..0ee4b8c97 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 11
 SUBLEVEL = 0
-EXTRAVERSION = -rc7
+EXTRAVERSION = -rc7-thor_kernel
 NAME = Baby Opossum Posse
 
 # *DOCUMENTATION*
-- 
2.44.0

